packages/wp-story-editor/src/api/media.js

```diff
@@ -122,8 +126,8 @@ export async function getOptimizedMediaById(config, mediaId) {
 
   const result = await apiFetch({ path });
 
-  if (result?.meta?.web_stories_optimized_id) {
-    return getMediaById(config, result.meta.web_stories_optimized_id);
+  if (result?.meta?.webStoriesOptimizedId) {
```

Comment:
Same here, I don't think the result we get from the REST API is camel-cased already.

Reply:
Updated in  https://github.com/google/web-stories-wp/pull/9683/commits/c29543c002a5f6c407e0d8f56b2d878b44ae7acf